src/ingester/persist/mod.rs

```diff
@@ -358,6 +367,32 @@ async fn execute_account_update_query_and_update_balances(
     Ok(())
 }
 
+async fn append_addresses(
```

Comment:
```suggestion
async fn insert_addresses_into_queues(
```
This inserts into the address queue right?

Reply:
or append_addresses_to_queues